java/src/test/java/com/google/oak/crypto/EncryptorTest.java

```diff
@@ -17,90 +17,82 @@
 package com.google.oak.crypto;
 
 import com.google.oak.crypto.hpke.KeyPair;
-import com.google.oak.crypto.v1.AeadEncryptedMessage;
 import com.google.oak.crypto.v1.EncryptedRequest;
 import com.google.oak.crypto.v1.EncryptedResponse;
 import com.google.oak.util.Result;
-import com.google.protobuf.ByteString;
-import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-import java.util.Optional;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
 public class EncryptorTest {
-  private static final byte[] TEST_REQUEST_PLAINTEXT =
-      new byte[] {'R', 'e', 'q', 'u', 'e', 's', 't'};
-  private static final byte[] TEST_REQUEST_ASSOCIATED_DATA = new byte[] {'d', 'a', 't', 'a', '1'};
-  private static final byte[] TEST_RESPONSE_PLAINTEXT =
-      new byte[] {'R', 'e', 's', 'p', 'o', 'n', 's', 'e'};
-  private static final byte[] TEST_RESPONSE_ASSOCIATED_DATA = new byte[] {'d', 'a', 't', 'a', '2'};
+    private static final byte[] TEST_REQUEST_PLAINTEXT = new byte[] { 'R', 'e', 'q', 'u', 'e', 's', 't' };
```

Comment:
Yes, "Request".getBytes(Charsets.US_ASCII) works, however it pulls in a dependency on charsets which I feel neutralizes the gain in readability. Leaving for now, also since out of scope of this PR.